service/src/test/java/bio/terra/workspace/service/workspace/ApplicationServiceTest.java

```diff
@@ -108,6 +130,7 @@ void setup() throws Exception {
                 Mockito.any(), Mockito.eq(SamResource.WORKSPACE), Mockito.any(), Mockito.any()))
         .thenReturn(true);
 
+    appService.enableTestMode();
```

Comment:
I wonder if there's a Springier way of achieving this, like injecting a fake version of the service, or something. It's a bit odd having a test method in the production code.

Reply:
There is probably a better way. Not salient for this PR, though.